### DISCUSSION TITLE
Playback status on TV

### DISCUSSION THREAD

**Discussion by DraculaVK** [ExperienceLovelace/ha-floorplan#128] [Q&A]
Hi, should I have a question going somehow to get the PNG image loaded? I wanted to make the TV show me in the floor plan what is currently playing on the TV, but Entity {entity.attributes.entity_picture} returns a PNG image. Or did someone solve something similar?
Thank you very much to the developers.

**Comment by pkozul** [comment]
Hi @DraculaVK 

According to the SVG specs, PNG images should be supported.

https://developer.mozilla.org/en-US/docs/Web/SVG/Element/image

I've just checked my setup, and it seems my entities only contain JPEG images. I will do some tests to see what happens when PNG images are used instead.

WIll keep you posted.

**Reply by OzGav** [reply]
Petar is this at all related to this old issue https://community.home-assistant.io/t/video-on-floorplan/27743/31?u=ozgav ? I know this is referring to the old version of floorplan but was just wondering?

**Comment by DraculaVK** [comment]
Hi, I came up with the problem that I used the correct object in the SVG floor plan. I used accidentally closed curves and not a rectangle drawing. Now everything works to help me see how the camera is made.

**Reply by exetico** [reply]
You can help other people, by sharing your SVG and Code 😊👍.

Share it if you like, and remember to select an answer for your question.

Have a great weekend.

**Reply by DraculaVK** [reply]
I am attaching the code. So far working version. Used a pair of sensors and a camera. I want to expand over time. Now I'm dealing with bitmap sizes. Otherwise great job. 
[Home.zip](https://github.com/ExperienceLovelace/ha-floorplan/files/6795778/Home.zip)
